### ISSUE TITLE
Player chat commands

### ISSUE THREAD

**Issue by zrisher** [stackcollision/GardenConquest#14]
Upon adding this mod to an existing world, most players will probably have at least 2-3 grids that are being derelicted somewhere far away on the map that they had no idea about / haven't seen  in days. Lost ships, debree, etc - there's a ton of this. Even if the mod's been on for a while, subsequent to battles there will be a ton of random stuff on this list.

I expect all these extra derelicts will give players a bunch of notifications. Plus, if losing a ship players might want some info on it. 

I propose a chat command to list the player's derelicting grids - their name and position - and optionally remove them individually. 

This is low priority.

**Comment by stackcollision** [comment]
Position could get hairy.  What happens if I steal someone else's ship?

**Comment by zrisher** [comment]
That's a great question. Some of the controls I was hoping to give players - list ship class counts, names of ships in class, etc - seem pretty safe to just give to the BigOwner's faction. Others are more complex:
- ability to forcibly delete the license block on a ship and start a dereliction timer - This is needed to make it so players don't have to fly halfway across the map to disown a ship they'd rather just trash that's counting against their limits. If you capture a ship and the owner starts a Dereliction, you still have 2 hours (or whatever the default is) to get it home and secure it. And if we implemented #10 with inventory, you would still have access to those License blocks. So this seems safe enough.
- ability to see the position of the grid - This is really tricky. It would be so helpful for player-driven cleanup and understanding, but you can't do it if someone has stolen the ship. How do we know someone has stolen the ship? 
  - We could base it off control of the License Block, but those could be hidden in the bowls of a ship and make stealing them both really difficult and incredibly risky.
  - What if we marked a grid as `stolen` if there was a primary-controlling cockpit (remember they introduced this a few patches ago) added to the grid that is owned by a faction != the BigOwner faction? Then you could check against that flag before displaying position. That's exactly what's needed to steal it by flying it away. But what if you just tugged it away with landing gear? Then there's literally no way for us to know we shouldn't give away its position.
  - **It seems like the best solution might just be to not display positions. If a faction has lost a ship and it's counting against their class limits, they can just force the dereliction timer.**

**Comment by zrisher** [comment]
"/gc fleet" and "/gc violations" now implemented. These seem to be the major holes remaining:

`/gc fleet disown [partialdisplayname/partialentityid]`
`/gc admin fleets`

These need some info from the server, but only once and then they can cache it:
`/gc help classes`
`/gc help cps`

**Comment by zrisher** [comment]
There needs to be a chat command that outlays how to get started as a new player. Five people have complained that the initial "getting started" info is missing and super hard to figure out.

**Comment by zrisher** [comment]
Remaining to do:
- /gc fleet disown [partialdisplayname/partialentityid]
- /gc admin fleets
- show locations of fully-owned ships with /gc fleet

**Comment by zrisher** [comment]
Updated remaining to do:
- /gc fleet
  - show ship list numbers for use with commands
  - show ship positions
- /gc fleet disown [ship name/ number]
- /gc fleet stop [ship name/ number] - remove overrides, turn on dampners, turn off grav drive
- /gc admin fleets
- /gc classes alias for /gc help classes

**Comment by zrisher** [comment]
#94 gave us 
- /gc fleet
  - show ship list numbers for use with commands
  - show ship positions

Now we just need:
- /gc fleet disown [ship name/ number]
- /gc fleet stop [ship name/ number] - remove overrides, turn on dampners, turn off grav drive
- /gc admin fleets
- /gc classes alias for /gc help classes

**Comment by zrisher** [comment]
#103 gave us 
- /gc fleet stop [ship name/ number] - remove overrides, turn on dampners, turn off grav drive

Now we just need:
- /gc fleet disown [ship name/ number]
- /gc admin fleets
- /gc classes alias for /gc help classes

**Comment by zrisher** [comment]
`/gc classes alias` for `/gc help classes` is done.

Now we just need the below. We'll open separate issues for each of these.
- /gc fleet disown [ship name/ number] - #125
- /gc admin fleets - #126
